tests/suites/helpers.function

```diff
@@ -423,3 +432,11 @@ static void test_fail( const char *test, int line_no, const char* filename )
     test_info.line_no = line_no;
     test_info.filename = filename;
 }
+
+static void test_skip( const char *test, int line_no, const char* filename )
+{
+    test_info.failed = 2;
```

Comment:
Ugh. `failed` is no longer a boolean stored in an `int`. Please change `failed` to an enum with constants for pass, fail, skip.

Not a blocker because we're in rush mode but note that this needs to be fixed.

Reply:
Agreed, and changed in 262c2ee.